src/components/GeoCard.jsx

```diff
@@ -1,199 +1,81 @@
 import 'leaflet/dist/leaflet.css'
 
-import Box from '@material-ui/core/Box'
-import Skeleton from '@material-ui/lab/Skeleton'
-
-import isSameDay from 'date-fns/is_same_day'
-import isSameYear from 'date-fns/is_same_year'
+import { useTheme } from '@material-ui/core/styles'
 
-import Icon from 'cozy-ui/transpiled/react/Icon'
-import ClockIcon from 'cozy-ui/transpiled/react/Icons/Clock'
-import CompassIcon from 'cozy-ui/transpiled/react/Icons/Compass'
-import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
-import Typography from 'cozy-ui/transpiled/react/Typography'
-import FlagIcon from 'cozy-ui/transpiled/react/Icons/Flag'
-import { useI18n } from 'cozy-ui/transpiled/react/I18n'
-import Stack from 'cozy-ui/transpiled/react/Stack'
-
-import {
-  getStartPlaceDisplayName,
-  getEndPlaceDisplayName,
-  getFormattedDuration,
-  getModes
-} from 'src/lib/trips'
+import './styles.styl'
 
+// use http://leaflet-extras.github.io/leaflet-providers/preview/ to choose a tileLayer
 const setupMap = node => {
   const map = L.map(node).setView([51.505, -0.09], 13)
 
-  L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-    attribution:
-      '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-  }).addTo(map)
+  L.tileLayer(
+    'https://{s}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}{r}.png',
+    {
+      attribution:
+        '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors &copy; <a href="https://carto.com/attributions">CARTO</a>'
+    }
+  ).addTo(map)
 
   return map
 }
 
-const mapStyle = {
-  height: 400
-}
-
-const POMEGRANATE = '#FF0017'
-const EMERALD = '#00D35A'
-const SLATE_GREY = '#313640'
-
-const geojsonMarkerOptions = {
-  radius: 6,
-  color: SLATE_GREY,
-  weight: 1,
-  opacity: 1,
-  fillOpacity: 0.8
-}
-
-const pointToLayer = (feature, latlng) => {
-  return L.circleMarker(latlng, {
-    ...geojsonMarkerOptions,
-    fillColor:
-      feature.properties.feature_type == 'end_place' ? POMEGRANATE : EMERALD
-  })
-}
+const makeGeoJsonOptions = theme => ({
+  style: feature => {
+    return {
+      weight: feature.properties.feature_type === 'section' ? 5 : 2,
+      color: theme.palette.primary.main
+    }
+  },
+  pointToLayer: (feature, latlng) => {
+    return L.marker(latlng, {
+      icon: L.divIcon({
+        html: `<div></div>`,
+        iconSize: [12, 12],
+        className: `cozy-leaflet-markers ${
+          feature.properties.feature_type === 'end_place'
+            ? 'cozy-leaflet-markers-end'
+            : 'cozy-leaflet-markers-start'
+        }`
+      })
+    })
+  }
+})
 
-const geojsonOptions = { pointToLayer }
+const makeMapStyles = ({ toolbarHeight }) => ({
+  mapContainer: {
+    height: `calc(100vh - ${toolbarHeight}px - var(--sidebarHeight))`
+  }
+})
 
 const TripsMap = ({ trip }) => {
   const nodeRef = useRef()
+  const toolbarHeight = document.getElementById('coz-bar').offsetHeight
```

Comment:
nit: I don't know if I do not prefer to have a CSS var for this one instead of relying on dom manipulation.

Reply:
L'approche DOM est plus juste non ? (via une var il faudrait la setter en dur ? donc si on modifie la barre ou si elle a une comportement différent selon les cas, la valeur settée peut ne pas être la bonne ?)

Reply:
Elle a une valeur en dure pour la hauteur de cette bar. Mais tout devrait se baser sur cette variable. Si on peut éviter de manipuler le DOM on se doit de le faire